src/components/ItemCard.js

```diff
@@ -0,0 +1,8 @@
+export default function ItemCard() {
+  return (
+    <li className="Item-card">
```

Comment:
great to put the li element inside the component. looks super clean in the main.js now. :)))

Reply:
Thanks John, trying to follow your approach.